content/en/platform/corda/5.0/application-networks/creating/mgm/register.md

```diff
@@ -152,4 +153,23 @@ Invoke-RestMethod -SkipCertificateCheck  -Headers @{Authorization=("Basic {0}" -
 ```
 {{% /tab %}}
 {{< /tabs >}}
-If successful, you should see the `APPROVED` registration status.
+
+The returned status of the registration request will be one of the following options:
+
```

Comment:
Some descriptions say "submitted via the REST API" . Does this not apply to them all? If so, is it necessary? Is it not implied by the intro sentence before the bullets?

Reply:
@nadinequinr3 no, it doesn't apply to all statuses. Here's an explanation from @charlieR3 :
My intention was to differentiate between 1) the registration request the user submits through the REST API and 2) the registration request that the member’s cluster sends to the mgm.
The request sent to the mgm will include the information the user submitted in the rest API plus additional information added by the platform.